godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch

```diff
@@ -3,15 +3,15 @@
   <!-- arguments -->
   <arg name="sim_robot" default="true"/>
   <arg name="robot_ip" unless="$(arg sim_robot)" />
-  <arg name="sim_sensor" default="true"/>
   <arg name="sim_laser" default="true"/>
   <arg name="laser_ip" unless="$(arg sim_laser)" default="192.168.32.50"/>
   <arg name="robot_model_plugin" default="abb_irb2400_descartes/AbbIrb2400RobotModel"/>
+  <arg name="sim_sensor" default="true"/>
+  <arg name="real_pcd" default="false"/>
+  <arg name="pcd_location"/> <!-- path to pcd file of part -->
```

Comment:
This has got to be a behavior change from previous versions of ROS - I thought roslaunch was quite strict about providing parameters to all of the arguments of a launch file.